### PR TITLE
[Bug fix] Create local sqlite directory at init

### PR DESCRIPTION
**Pull request by deven298**
as title suggests

### CODE REVIEW

File: embedchain/memory/base.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
                 created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
                 PRIMARY KEY (id, app_id)
             )
-            """
+"""
```
Comment (deshraj) [inline embedchain/memory/base.py:21]: This is probably not needed.

File: embedchain/embedchain.py [hunk 2]
```diff
@@ -50,6 +51,8 @@ def __init__(
         :type system_prompt: Optional[str], optional
         :raises ValueError: No database or embedder provided.
         """
+        # Setup user directory if it doesn't exist already
+        Client.setup_dir()
```
Comment (deshraj) [inline embedchain/embedchain.py:55]: I think this needs to be outside the Embedchain class to be able to run when we do `import embedchain`

File: embedchain/memory/base.py (hunk 1 above)
Comment (deven298) [reply embedchain/memory/base.py:21]: lets just keep it. Its more cleaner

**Comment by deshraj** [comment]
Closing this since already fixed in #975

**Pull request closed without merging**